docs/Encoder.md

```diff
@@ -1,57 +1,103 @@
 # AMDC Encoder Input
 
-This document describes the design considerations and implementation details for the encoder input subsystem on the Advanced Motor Drive Controller (AMDC). A block diagram is presented and each component discussed in detail. Specifications of each component are provided based on the datasheet.
+This document describes the design considerations and implementation details for the encoder input subsystem on the Advanced Motor Drive Controller (AMDC). A block diagram is presented and each component is discussed in detail. Specifications of each component are provided based on the datasheet.
+
+## Relevant Versions of AMDC Hardware
+
+AMDC REV D
 
 ## Design Requirements
 
-There are several design requirements while designing encoder subsystem of AMDC:
-1.	To be able to operate with many standard incremental encoders.
-2.	Inner compatibility between chips: differential to single-ended converter chip output should be compatible with level shifter chip input.
-3.	Level shifter reduces signal voltage levels enough so that they can be transmitted to PicoZed.
+There are two design requirements that must be met for the encoder subsystem of the AMDC:
+
+1. Interface standard differential incremental encoders (5V differential unipolar signals) to FPGA.
+
+2. Provide a high-bandwidth front-end so that the encoder line count is limited by the FPGA speed, not circuitry.
+
 
 ## Block Diagram
 
 <img src="images/amdc-encoder.svg" />

```

Comment:
Please remove the repeated blocks for encoder 2. Instead indicate `2x copies on PCB` with a blue box around it, like you are doing in PR #109.